docs/environment.yaml

```diff
@@ -2,7 +2,8 @@ name: snakepipes
 channels:
   - conda-forge
 dependencies:
- - python >=3.6
+ - python >=3.9
  - pyyaml >= 5.1
  - sphinx-argparse
+ - jinja2 < 3.1.0
```

Comment:
same.

Reply:
it looks like you're viewing an older commit here, one that I have reversed.